buildtest/buildsystem/base.py

```diff
@@ -105,24 +105,24 @@ def __repr__(self):
     # Validation
 
     def _validate(self):
-        """Given a loaded recipe, validate that the type is known in the lookup
+        """Given a loaded Buildspec recipe, validate that the type is known in the lookup
            to buildtest. If a version is provided, honor it. If not, use latest.
            We also don't allow repeated keys in the same file.
         """
 
```

Comment:
The only variable I think we should keep constant and not change is recipe - it is implied that recipe for a build config that loads a buildspec file Is a build spec recipe. Making the variable this long isn’t as clean and nice looking :-)

Reply:
Yea i agree, i like the word ``recipe`` its the loaded Buildspec i can revert this.